src/components/layouts/pages/inspect/pdfDocument/PdfDocument.js

```diff
@@ -120,7 +120,7 @@ class PdfDocument extends React.PureComponent {
                 ? this.state.errorsRects[prevProps.selectedCheck]?.pageIndex
                 : this.state.mapOfErrors[this.props.selectedCheck]?.pageIndex;
 
-            if (prevPage) {
+            if (!_.isNaN(prevPage)) {
```

Comment:
This check doesn't make sence. Possible invalid values for `prevPage` are `undefined` and `-1` , but not `NaN`